src/main/resources/templates/index.html

```diff
@@ -1,8 +1,27 @@
 <!DOCTYPE html>
-<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
-    <head th:replace=" head :: head">
-    </head>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/html">
+<head>
```

Comment:
we moved out all head related stuff to head.html and included it in other pages, this is older and will therefore break the frontend